src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -1281,10 +1282,15 @@ static int TestMain()
             {
                 CompilationVerifier verifier = CompileWithNoFramework(source + InstrumentationHelperSource, emitOptions: EmitOptions.Default.WithInstrument("Test.Flag"));
             }
-            catch (SyntheticBoundNodeFactory.MissingPredefinedMember missing)
+            catch (EmitException exception)
             {
-                Assert.True(missing.Diagnostic.ToString().Contains("IntrospectionExtensions.GetTypeInfo"));
-                return;                
+                foreach (Diagnostic diagnostic in exception.Diagnostics)
+                {
+                    if (diagnostic.ToString().Contains("Missing compiler required member 'System.Reflection.IntrospectionExtensions.GetTypeInfo'"))
```

Comment:
Formatting diagnostics this way is sensitive to the current UI language. At least consider passing InvariantCulture to the ToString, but using VerifyDagnostics helper is preferred,